test/Libraries/RevitIntegrationTests/FloorTypeTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Autodesk.Revit.DB;
+using Dynamo.Nodes;
+using Autodesk.DesignScript.Geometry;
+using CoreNodeModels.Input;
+using NUnit.Framework;
+using RevitServices.Persistence;
+using RevitTestServices;
+using RTF.Framework;
+using Revit.Elements;
+
+namespace RevitSystemTests
+{
+    [TestFixture]
+    public class FloorTypeTests : RevitSystemTestBase
+    {
+        [Test]
+        [TestModel(@".\FloorType\FloorType.rvt")]
+        public void CanGetFloorTypeThermalProperties()
+        {
+            // Arrange
+            string samplePath = Path.Combine(workingDirectory, @".\FloorType\canGetFloorTypeThermalProperties.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+
+            double expectedWallTypeAbsorptance = 0.7;
+            double expectedWallTypeHeatTransferCoefficient = 0.117437;
+            double expectedWallTypeRoughness = 3;
+            double expectedWallTypeThermalMass = 365549.43;
+            double expectedWallTypeThermalResistance = 8.515159;
+
+            // Act
+            ViewModel.OpenCommand.Execute(testPath);
+            RunCurrentModel();
+
+            var resultWallTypeAbsorptance = GetPreviewValue("56cb939de2cc4ca4a21731ba78f4299b");
+            var resultWallTypeHeatTransferCoefficient = GetPreviewValue("6d63251353044032b5167428b7749f62");
+            var resultWallTypeRoughness = GetPreviewValue("9cd0f885729641dc928f93250bae096a");
+            var resultWallTypeThermalMass = GetPreviewValue("e60b5201ae23449196beb090d0d825d1");
+            var resultWallTypeThermalResistance = GetPreviewValue("1ca68872f565419b838cdbb8306057d3");
+
+            // Assert
+            Assert.AreEqual(expectedWallTypeAbsorptance, (double)resultWallTypeAbsorptance, 0.001);
```

Comment:
replace all instances of `0.001` with a `tolerance` `const` for this test class.
top tip - you have a `ctrl+.` intellisense refactor suggestion for this ;)